src/sync/installation.ts

```diff
@@ -454,65 +455,64 @@ export async function maybeScheduleNextTask(
 const redis = new IORedis(getRedisInfo("installations-in-progress"));
 
 const RETRY_DELAY_BASE_SEC = 60;
-export const processInstallation =
-	(app: Application) => {
-		const inProgressStorage = new RedisInProgressStorageWithTimeout(redis);
-		const deduplicator = new Deduplicator(
-			inProgressStorage, 1_000
-		);
-
-		return async (data: BackfillMessagePayload, sentry: Hub, logger: Logger): Promise<void> => {
-			const { installationId, jiraHost } = data;
-
-			logger.child({ gitHubInstallationId: installationId, jiraHost });
-
-			try {
-				if (await isBlocked(installationId, logger)) {
-					logger.warn("blocking installation job");
-					return;
-				}
+export const processInstallation = (app: Application) => {
```

Comment:
There's a big `git diff` here. It'd be great if you can stick to original format and show a cleaner `git diff` .